server/src/main/scala/org/http4s/server/middleware/authentication/DigestUtil.scala

```diff
@@ -30,6 +27,7 @@ private[authentication] object DigestUtil {
     * @param qop
     * @return
     */
+  @deprecated("Use org.http4s.middleware.authentication.DigestUtil instead", "0.21")
```

Comment:
I expect this to be part of 0.21.7.  We can be more specific in the tag.

Reply:
updated